apps/src/code-studio/components/progress/ScriptTeacherPanel.jsx

```diff
@@ -69,21 +78,52 @@ class ScriptTeacherPanel extends React.Component {
       students
     } = this.props;
 
+    const inMiniRubricExperiment = experiments.isEnabled(
+      experiments.MINI_RUBRIC_2019
+    );
+
+    const currentLevelSection = sectionData ? sectionData.section_levels : null;
+
+    const currentStudent = sectionData
+      ? sectionData.section.students.find(
+          student => this.props.getSelectedUserId() === student.id
+        )
+      : null;
+    const currentStudentLevel = sectionData
+      ? sectionData.section_levels.find(
+          level => this.props.getSelectedUserId() === level.user_id
```

Comment:
I think you've protected against this later, but I think it would be a bit safer to change the first part of this to something like `const currentStudentLevel = sectionData && currentStudent ? ...` so that you don't unintentionally set currentStudentLevel when there is no student.